profiles/car.lua

```diff
@@ -261,7 +261,7 @@ function way_function (way, result)
     -- also respect user-preference for HOV-only ways when all lanes are HOV-designated
     local function has_all_designated_hov_lanes(lanes)
       local all = true
-      for lane in lanes:gmatch("(%w+)") do
+      for lane in (lanes .. '|'):gmatch("([^|]*)|") do
```

Comment:
Gnaaa this is such a bad hack - can you please make a comment at least, I know Lua is horrible. Then you can merge it into master asap.